### DISCUSSION TITLE
Fetching uptream's new changes

### DISCUSSION THREAD

**Discussion by davidandreoletti** [Mystfit/ESP32-BLE-CompositeHID#17] [General]
Hi @Mystfit ,

I would like to use your project for its Xinput support yet need[ this commit](https://github.com/Mystfit/ESP32-BLE-CompositeHID/commit/dbdd1a03e27bddb1f8b856f3141b8bad72b43418) from upstream. The commit is not currently inside your repo.

Do you plan to bring upstream's new changes into this repo soon ?

**Comment by Mystfit** [comment]
@davidandreoletti Merged in https://github.com/Mystfit/ESP32-BLE-CompositeHID/commit/8e105d7af0cc018a3acf5943ab23fba515d99271
